mta.yaml

```diff
@@ -82,7 +82,7 @@ resources:
       service: postgresql-db
       service-plan: standard
       config:
-        engine_version: "14"
+        engine_version: "16"
```

Comment:
We might need to re-create test Postgres instance after that.

Reply:
yes, that could be the case. Last time if I remember correct the instance was updated during deployment. We will see.

Reply:
will downgrade to 15 in a new PR.